### DISCUSSION TITLE
App translation

### DISCUSSION THREAD

**Discussion by Gusti23** [on3iro/aeons-end-randomizer#490] [Ideas]
Is it possible to make translation of full app instead of content. I mean to translate menu, expedition and other stuff?

**Comment by on3iro** [comment]
@Gusti23 it currently is not, however @jannydiamond was recently thinking about making this possible and adding a translation library.

**Reply by Gusti23** [reply]
If it will be possible just ping me and I will start working on the translation. Meanwhile I am close to make a pull request with PL translation of New Age & Into the Wild. For now all expansion in polish language will be translated in the app.
